diff --git a/Makefile b/Makefile
index b0a9f1c..5e3febd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
-GPU=0
-CUDNN=0
+GPU=1
+CUDNN=1
 OPENCV=0
 OPENMP=0
-DEBUG=0
+DEBUG=1
 
 ARCH= -gencode arch=compute_20,code=[sm_20,sm_21] \
       -gencode arch=compute_30,code=sm_30 \
diff --git a/examples/rnn.c b/examples/rnn.c
index 45e8deb..3cb71c5 100644
--- a/examples/rnn.c
+++ b/examples/rnn.c
@@ -128,7 +128,7 @@ void train_char_rnn(char *cfgfile, char *weightfile, char *filename, int clear,
         fclose(fp);
     }
 
-    char *backup_directory = "/home/pjreddie/backup/";
+    char *backup_directory = "/home/alex/proverbot9001/darknet/backup";
     char *base = basecfg(cfgfile);
     fprintf(stderr, "%s\n", base);
     float avg_loss = -1;
diff --git a/python/proverbot.py b/python/proverbot.py
index 095aae8..1bbb5cc 100644
--- a/python/proverbot.py
+++ b/python/proverbot.py
@@ -32,6 +32,6 @@ def predict_tactics(net, s, n):
     tacs = sorted(tacs, key=lambda x: -x[1])
     return tacs
 
-net = load_net("cfg/coq.test.cfg", "/home/pjreddie/backup/coq.backup", 0)
-t = predict_tactics(net, "+++++\n", 10)
-print t
+# net = load_net("cfg/coq.test.cfg", "/home/pjreddie/backup/coq.backup", 0)
+# t = predict_tactics(net, "+++++\n", 10)
+# print t
